src/views/nunjucks/form.njk

```diff
@@ -0,0 +1,11 @@
+{% extends "./nunjucks/layout.njk" %}
+{% from "nunjucks/macros/forms/index.njk" import formRender, formErrorSummary %}
+{% from "nunjucks/macros/title.njk" import title %}
+
+{% block content %}
+  <div class="govuk-grid-row">
```

Comment:
I think this needs an extra div with a column class to render with the correct margin at mobile sizes